aws/resource_aws_guardduty_detector.go

```diff
@@ -30,6 +30,11 @@ func resourceAwsGuardDutyDetector() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"finding_publishing_frequency": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "SIX_HOURS",
```

Comment:
Nit: We can use the AWS Go SDK provided constant here and add plan time validation for this attribute via:

```go
Default: guardduty.FindingPublishingFrequencySixHours,
ValidateFunc: validation.StringInSlice([]string{
  guardduty.FindingPublishingFrequencyFifteenMinutes,
  guardduty.FindingPublishingFrequencyOneHour,
  guardduty.FindingPublishingFrequencySixHours
}, false),
```